samples/cpp/benchmark_app/main.cpp

```diff
@@ -772,8 +789,11 @@ int main(int argc, char* argv[]) {
                 batchSize = get_batch_size(app_inputs_info.front());
 
                 slog::info << "Model batch size: " << batchSize << slog::endl;
-            } else if (batchSize == 0) {
-                batchSize = 1;
+            } else {
+                warn_no_batch_for_dynamic(app_inputs_info);
```

Comment:
Applicable for static also (previously warning was printed inside get_batch_size)

Reply:
Updated